runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.7.9
```

Comment:
We should also update the travis.yml file with this python version so that the versions remain consistent when testing before merge.

Reply:
Done. Thanks

Reply:
Awesome!